app/views/citizens/other_assets/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :navigation do %>
-  <%= link_to t('generic.back'), citizens_additional_accounts_path, class: 'govuk-back-link', id: 'back' %>
+  <%= link_to t('generic.back'), back_path, class: 'govuk-back-link', id: 'back' %>
```

Comment:
`link_to t('generic.back'), back_path, class: 'govuk-back-link', id: 'back'`

This code is repeated in at least 6 places. I'd think a helper method makes sense, along the lines of `provider_back_link`. In fact, if the two journeys used the same method name form the "Back url/path", the same helper method could be used across both journeys. It may even make sense to make the default behaviour of `content_for :navigation` to call this back link helper method.